doc/tech/usecases.md

```diff
@@ -11,3 +11,26 @@
 
 <a name="5.1"></a>
 ## 5.1 Edge
+
+3-Approach & Scope  : 
+
+Edge Approach:
+-	 All gaps under Reference Model , Reference Architecture 01 -OpenStack- , Reference Architecture 02 - K8s will be identified and Edge requirements will be fulfilled  under Reference Model , Reference Architecture 01 -OpenStack- , Reference Architecture 02 - K8s  
+
+Edge scope under CNTT will cover :
```

Comment:
Should the scope cover "Scale" -- edge locations are likely to be in thousands even hundreds of thousands and that has implications.